maintenance/wikia/sync_user_groups.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Wikia\DependencyInjection\Injector;
+use Wikia\UserGroups\Maintenance\UserGroupUpdater;
+
+$wgCommandLineSilentMode = true;
+
+require_once(dirname(__FILE__).'/../Maintenance.php');
+
+class SyncWikiUserGroups extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->addOption('wiki_id', '', true, true);
+	}
+
+	public function loadParamsAndArgs($self = null, $opts = null, $args = null) {
+		parent::loadParamsAndArgs($self, $opts, $args);
+
+		if (!empty($this->mOptions['wiki_id'])) {
+			putenv("SERVER_ID={$this->mOptions['wiki_id']}");
+		}
+	}
+
+	public function execute() {
+		global $wgCityId;
+
+		/** @var UserGroupUpdater $updater */
+		$updater = Injector::getInjector()->get(UserGroupUpdater::class);
+		$updater->updateForWiki($wgCityId, wfGetDB(DB_SLAVE));
```

Comment:
Additionally, classes that use `Injector` rather than the `$wg*` variables do not need to extend `WikiaBaseTest` at all, since it doesn't rely on any external state except for the `Injector`.